tests/links_tests/test_mlp_bn.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from builtins import *  # NOQA
+from future import standard_library
+standard_library.install_aliases()
+
+import unittest
+
+import numpy as np
```

Comment:
In other files, `numpy` and `chainer` are in the same group of the import order.